package-res/resources/web/pentaho/type/ComplexChangeset.js

```diff
@@ -1,159 +0,0 @@
-/*!
- * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Suggest changing to:

WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either expressed or implied.